KsApi/MockService.swift

```diff
@@ -1529,3 +1534,19 @@
     }
   }
 #endif
+
+private extension Result {
```

Comment:
We only use this in this file. Previously this extension was public. We could keep this or replace with a helper function 🤷‍♂